.travis.yml

```diff
@@ -19,17 +19,16 @@ install:
   - pip install flake8
   - make lint
   - pip install -r requirements_test.txt
+  # refresh the git submodule ./vendor/acs4_py
+  # refresh the git submodule ./vendor/infogami
+  # git commit those changes into this pull request
   # if Travis is running Python 3, then
-  #   refresh the git submodule ./vendor/acs4_py
-  #   refresh the git submodule ./vendor/infogami
-  #   git commit those changes into this pull request
-  #   TODO: flake8 ./vendor --select=E9,F63,F7,F82 to unsure new versions pass
   #   create a legacy Python for npm/node until nodejs/node#25789 is resolved
+  - pushd vendor/acs4_py  && git pull origin master && popd
+  - pushd vendor/infogami && git pull origin master && popd
```

Comment:
I'm not sure about this method to update the required versions of our submodules for python3. It looks like this is only happening on Travis so there is no mechanism to checkout the correct code in any other environment. The code SHA/tag is not fixed either, like it is with the standard gitmodules in `vendor` https://github.com/internetarchive/openlibrary/tree/master/vendor

Is there a way we can leverage gitmodules to check out two different points in infogami and acs4 depending on the Python version that applies in all cases, using the existing gitmodules mechanism? (I have not begun to investigate this yet)

Reply:
acs4_py can be removed.  It is already 💯 ready for Python 3.